GitUI/CommandsDialogs/FormResolveConflicts.cs

```diff
@@ -1243,6 +1256,28 @@ private void OpenMergetool_Click(object sender, EventArgs e)
             OpenMergeTool();
         }
 
+        private void customMergetool_Click(object sender, EventArgs e)
+        {
+            if (sender == customMergetool)
+            {
+                // Cancel dropdown manually if "main menu" is selected
+                customMergetool.HideDropDown();
+                ConflictedFilesContextMenu.Hide();
+            }
+
+            string customTool = (sender as ToolStripMenuItem)?.Tag as string;
```

Comment:
Should we assert it is not null or empty?
```suggestion
            string customTool = (sender as ToolStripMenuItem)?.Tag as string;
            Debug.Assert(!string.IsNullOrWhitespace(customTool));
```

Reply:
null and empty is OK, see comment to mstv.